tools/README.md

```diff
@@ -19,58 +19,47 @@ The tool allows developers to publish/consume messages to/from queues and topics
 
 ### Publish messages to queues/topics
 
-1. Include the following details in broker_performance_test_publisher.properties file which is located at resources/.
+1.Include the following details in broker_test_plan.properties file which is located at resources/.
```

Comment:
A space needs to be there after the period. Shall we correct in all places?

Reply:
Done